src/index.html

```diff
@@ -2,16 +2,237 @@
+            <label for="password">
+              Password:
+            </label>
+            <input
+              type="password"
+              id="password"
+              name="password"
+              minlength="8"
+              maxlength="18"
+              autocomplete="off"
+            >
+          </section>
+        </fieldset>
+
+        <fieldset class="group">
+          <legend>
+            An interesting fact about you!
+          </legend>
+
+          <section class="section">
+            Do you love cats?
+            <input
+              type="radio"
+              name="isLoveCat"
+              id="loveCat"
+            >
+            <label for="loveCat">
+              Yes
+            </label>
+
+            <input
+              type="radio"
+              name="isLoveCat"
+              id="notLoveCat"
+            >
+            <label for="notLoveCat">
+              No
+            </label>
+          </section>
+
+          <section class="section">
+            <label for="favColor">
+              What is your favorite color?
+            </label>
+            <input
+              type="color"
+              name="favColor"
+              id="favColor"
+            >
+          </section>
+
+          <section class="section">
+            <label for="isTimeGoToBed">
+              What time you go to bed?
+            </label>
+            <input
+              type="time"
+              name="isTimeGoToBed"
+              id="isTimeGoToBed"
+            >
+          </section>
+
+          <section class="section">
+            <label for="isFavBrandOfCars">
+              What are your favorite brand of cars?
+            </label>
+            <select
+              id="isFavBrandOfCars"
+              name="isFavBrandOfCars"
+              multiple
+            >
+              <option value="BMW">BMW</option>
+              <option value="Audi">Audi</option>
+              <option value="Lada">Lada</option>
+            </select>
+          </section>
+
+          <section class="section">
+            <label for="rateOurWork">
+              How do you rate our work?
+            </label>
+            <input
+              type="range"
+              name="rateOurWork"
+              id="rateOurWork"
+            >
+            </input>
+          </section>
+
+        </fieldset>
+
+        <fieldset class="group">
+          <legend>
+            Additional Info:
+          </legend>
+          <section class="section">
+            <label for="comments">
+              Comments:
+            </label>
+            <textarea
+              autocomplete="off"
+              name="comments"
+              id="comments"
+           ></textarea>
+          </section>
+
+          <section class="section">
+            <label for="recommendUs">
+              Would you recommend us?:
+            </label>
+            <select name="recommendUs">
+              <option value="true">Yes</option>
```

Comment:
Here should be second option with 'No' value

Reply:
Done